project/tagging/factories.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+from django.contrib.contenttypes.models import ContentType
+from factory import DjangoModelFactory, Faker, LazyAttribute
+from .models import CustomTag, TaggedItems
+
+
+class CustomTagFactory(DjangoModelFactory):
+    guid = LazyAttribute(lambda obj: uuid.uuid1())
+    name = Faker("word")
```

Comment:
We don't need to fake slug here - we'll create one on `save()` based on the name